### PR TITLE
docs: add BentoML to awesome-transformers

### PR DESCRIPTION
**Pull request by aarnphm**
# What does this PR do?

Kindly ask to add BentoML to the list of awesome projects that has transformers support

cc @parano

Signed-off-by: Aaron <29749331+aarnphm@users.noreply.github.com>

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

cc @sgugger

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: awesome-transformers.md [hunk 1]
```diff
@@ -231,6 +231,13 @@ Keywords: Data augmentation, Synthetic data generation, Audio, NLP
 
 Keywords: Stable-Diffusion, Blender
 
+## [BentoML](https://github.com/bentoml/BentoML)
+
+[BentoML](https://github.com/bentoml) is the unified framework for for building, shipping, and scaling production-ready AI applications incorporating traditional ML, pre-trained AI models, Generative and Large Language Models. 
+All Hugging Face Transformers models and pipelines can be seamlessly integrated into BentoML applications, enabling the running of models on the most suitable hardware and independent scaling based on usage.
```
Comment (ssheng) [inline awesome-transformers.md:237]:
```suggestion
All Hugging Face models and pipelines can be seamlessly integrated into BentoML applications, enabling the running of models on the most suitable hardware and independent scaling based on usage.
```

**Review by ssheng — APPROVED** [review]

**Comment by amyeroberts** [comment]
cc @LysandreJik

**Review by LysandreJik — COMMENTED** [review]
The addition looks good to me! Would it be possible to add it at the bottom of the file? This wasn't necessarily done for past PRs, I'll open a PR to ensure that it is done chronologically. Thanks.

**Comment by aarnphm** [comment]
I have updated the docs to the bottom of the page.

**Review by LysandreJik — APPROVED** [review]
Thank you! Merging it

**Pull request merged**